routes/api/asset/index.js

```diff
@@ -90,10 +90,19 @@ router.put('/:asset_id/settings', (req, res, next) => {
 });
 
 router.put('/:asset_id/status', (req, res, next) => {
-  // Update the asset status
+  const id = req.params.asset_id;
+  const {
+    closedAt = null,
+    closedMessage = null
+  } = req.body;
   Asset
-    .update({id: req.params.asset_id}, {status: req.query.status})
-    .then(() => res.status(204).end())
+    .update(id, {
+      closedAt: closedAt,
+      closedMessage: closedMessage
```

Comment:
So closed messages are set by asset and then default to the global message if none is set?